database/gormdb_assignment_test.go

```diff
@@ -48,26 +47,27 @@ func TestGormDBCreateAssignment(t *testing.T) {
 	admin := qtest.CreateFakeUser(t, db, 10)
 	qtest.CreateCourse(t, db, admin, &pb.Course{})
 
-	assignment := pb.Assignment{
+	gotAssignment := pb.Assignment{
 		CourseID: 1,
 		Order:    1,
 	}
 
-	if err := db.CreateAssignment(&assignment); err != nil {
+	if err := db.CreateAssignment(&gotAssignment); err != nil {
 		t.Fatal(err)
 	}
 
 	assignments, err := db.GetAssignmentsByCourse(1, false)
 	if err != nil {
 		t.Fatal(err)
 	}
+	wantAssignment := assignments[0]
 
 	if len(assignments) != 1 {
 		t.Fatalf("have size %v wanted %v", len(assignments), 1)
 	}
 
-	if !reflect.DeepEqual(assignments[0], &assignment) {
-		t.Fatalf("want %v have %v", assignments[0], &assignment)
+	if diff := cmp.Diff(wantAssignment, gotAssignment, protocmp.Transform()); diff != "" {
+		t.Errorf("CreateAssignment() or GetAssignmentsByCourse() mismatch (-wantAssignment, +gotAssignment):n%s", diff)
```

Comment:
This test function is testing `CreateAssignment()` and is only using `GetAssignmentsByCourse()` to get the `wantAssignment`. So please remove the `or GetAssign...`